### DISCUSSION TITLE
Disable Kafka clients metrics

### DISCUSSION THREAD

**Discussion by mihailmanoli** [open-telemetry/opentelemetry-java-instrumentation#7132] [Q&A] [answered]
Hi,
I'm using Dynatrace as backend storage for metrics and I need to disable Kafka clients metrics because those are consuming a lot of Dynatrace credits. 

I've added the following env variable to the auto-instrumented k8s deployment:
```
            - name: OTEL_INSTRUMENTATION_KAFKA_METRIC_REPORTER_ENABLED
              value: "false"
```
but I still observe that the Kafka metrics are pushed to the Dynatrace.

What I'm missing?

**Comment by trask** [comment, marked as answer]
hi @mihailmanoli! this setting looks correct to me. if you are still facing this, could you post a standalone repro that we can examine and troubleshoot?

**Reply by mihailmanoli** [reply]
hi @trask and thank you for the reply! 
I was looking at the Dynatrace credits consumed by Kafka metrics and I'm observing that everything is working as expected. Most probably the metrics were sent to Dynatrace by a service not having the mentioned env variable.
